backend/FileSystem/DirectoryManager.php

```diff
@@ -23,7 +26,7 @@ public function getApiDirectoryPath(string $relativePath): string
     {
         $directoryPath = $this->apiPath->getFull($relativePath);
         if (!file_exists($directoryPath)) {
-            mkdir($directoryPath, 0777, true);
+            mkdir($directoryPath, self::DIRECTORY_PERMISSIONS, self::RECURSIVE_CREATION);
```

Comment:
Try to use `static` instead `self`. It allow us to override constants in the child classes